File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace CustomDateFilterPopup {
    public partial class MainForm : DevExpress.XtraEditors.XtraForm {
        OrderList orderList;
        public MainForm() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            orderList = new OrderList(10);
            gridControl1.DataSource = orderList;
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace CustomDateFilterPopup
	Partial Public Class MainForm
		Inherits DevExpress.XtraEditors.XtraForm
		Private orderList As OrderList
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			orderList = New OrderList(10)
			gridControl1.DataSource = orderList
		End Sub
	End Class
End Namespace
